adapters/smaato/smaatotest/supplemental/bad-user-ext-req.json

```diff
@@ -0,0 +1,106 @@
+{
```

Comment:
There are 2 mockBidRequest values in this test too.

Reply:
done